### ISSUE TITLE
partial_fc, pytorch, embedding_size=128 , can't converge

### ISSUE THREAD

**Issue by meixitu17** [deepinsight/insightface#1370]
Hi,
    I use glint360K to train pytorch partial_fc, if the embedding_size=512, everything work fine.
    But if I changed embedding_size=128 , model name is iresnet34, the code can't converge, loss will drop to small  and nan quickly in one epoch.
    I use three GPU to train. batch_size=256, sample_rate=0.1

    Do I need change something if I changed embedding_size?

Thanks
meixitu

**Comment by nttstar** [comment]
Can you try a smaller learning rate?

**Comment by meixitu17** [comment]
Hi nttstar,
Thanks for your help.
Yes, I tried smaller learning rate, it can converge.

But I don't know why embedding_size=512 is ok

Thanks
meixitu

**Comment by nttstar** [comment]
Learning with 128 emb-size is 'harder'.

**Comment by meixitu17** [comment]
@nttstar, 

yes, it is reasonable.

Thanks
meixitu

**Comment by 22wei22** [comment]
> Hi nttstar,
> Thanks for your help.
> Yes, I tried smaller learning rate, it can converge.
> 
> But I don't know why embedding_size=512 is ok
> 
> Thanks
> meixitu

I meet the same problem. What is your lr rate and batchsize now?
